.github/workflows/main.yml

```diff
@@ -0,0 +1,66 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+group = "com.degitx.simulator"
```

Comment:
@IlyaMoskva we're using `org.cqfn.*` as a base for group id. I think we can use `org.cqfn.degitx.simulator` as a package name here

Reply:
@g4s8 Thanks for clarification, I'll use it.